controller/src/main/java/org/jboss/as/controller/audit/ManagedAuditLoggerImpl.java

```diff
@@ -528,6 +528,35 @@ public void updateInMemoryHandlerMaxHistory(String name, int maxHistory) {
         }
     }
 
+    public void copy(ManagedAuditLoggerImpl other) throws IOException {
```

Comment:
Yes, maybe using two references instead of switching would make it feel less hacky and more straightforward.